pupil_src/shared_modules/calibration_routines/gaze_mappers.py

```diff
@@ -79,9 +79,14 @@ def __init__(self, g_pool):
 
         self.min_pupil_confidence = 0.6
         self._caches = (deque(), deque())
-        self.temportal_cutoff = 0.3
+        self.default_temportal_cutoff = 0.3
         self.sample_cutoff = 10
 
+    def calculate_temporal_cutoff(cache):
+        if len(cache) < 2:
+            return self.default_temportal_cutoff
+        return np.mean(np.diff([d['timestamp'] for d in cache]))
+
```

Comment:
This means we aim at only matching frames that lie as close together as our average fps, right?
Shouldn't we at least leave a bit room, like maybe 2* the avg?
Because otherwise a single low-confidence value means already that we don't map binocular, right?
I feel like we actually want to do some fitting to other close samples if there's some low-conf involved.
But please correct me, I'm not in the loop on that dicussion.